### PR TITLE
Update duckduckgo.po fa_IR translation

### PR DESCRIPTION
**Pull request by mskf1383**
DuckDuckGo --> داک‌داک‌گو

### CODE REVIEW

File: locales/fa_IR/LC_MESSAGES/duckduckgo.po
Comment (GioSensation) [inline locales/fa_IR/LC_MESSAGES/duckduckgo.po:98]: The `msgid` shouldn't be changed. Reverting.

```suggestion
msgid ""
```

File: locales/fa_IR/LC_MESSAGES/duckduckgo.po
Comment (GioSensation) [inline locales/fa_IR/LC_MESSAGES/duckduckgo.po:102]: Add `fuzzy` to allow translated trademark.

```suggestion

#  Marked fuzzy because of translated trademark
#, fuzzy
```

File: locales/fa_IR/LC_MESSAGES/duckduckgo.po
Comment (GioSensation) [inline locales/fa_IR/LC_MESSAGES/duckduckgo.po:4502]: Adding back placeholder tags.

```suggestion
msgstr "%sما کمکتان می‌کنیم.%s از ردیابی شدن آنلاین خسته شده‌اید؟."
```

**Review by GioSensation — APPROVED** [review]

File: locales/fa_IR/LC_MESSAGES/duckduckgo.po
Comment (GioSensation) [reply locales/fa_IR/LC_MESSAGES/duckduckgo.po:4502]: @mskf1383 This string was missing placeholder tags and it breaks our release system. I've added them like this, but I'm not sure it's correct in your language. I am committing to see whether it makes our system happy, but I'll hold off merging until you've taken confirmed this is fine. Thanks for looking into this.

**Comment by GioSensation** [comment]
Thanks for your contribution, @mskf1383. Before merging, I'd like you to please double check [this comment](https://github.com/duckduckgo/duckduckgo-locales/pull/411#discussion_r552475798). Thanks!

**Comment by mskf1383** [comment]
Thank you for resolving my problems.
I correct that you said and you can merge it.
Thanks!

**Comment by GioSensation** [comment]
Thanks for the quick turnaround @mskf1383! Merging. 🎉

**Pull request merged**
